src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -6,64 +6,196 @@
 #
 import random as _random
 import turicreate as _tc
+import turicreate.toolkits._internal_utils as _tkutl
 from turicreate import extensions as _extensions
 from turicreate.toolkits._model import CustomModel as _CustomModel
+from turicreate.toolkits._model import PythonProxy as _PythonProxy
+from turicreate.toolkits.object_detector.object_detector import ObjectDetector as _ObjectDetector
 
 def create(dataset,
            target,
+           augmentation_mode="auto",
+           flip_horizontal=True,
+           flip_vertical=True,
            backgrounds=None,
-           feature=None,
            batch_size=0,
            max_iterations=0,
            seed=None,
-           verbose=True):
-    model = _extensions.one_shot_object_detector()
+           verbose=True,
+           **kwargs):
+    """
+    Create a :class:`ObjectDetector` model.
+
+    Parameters
+    ----------
+
+    dataset : SFrame | tc.Image
+        An SFrame that contains all the starter images along with their
+        corresponding labels.
+        If the dataset is a single tc.Image, this parameter is just that image.
+        Every starter image should entirely be just the starter image without
+        any padding.
+        RGB and RGBA images allowed.
+
+    target : string
+        The target column name in the SFrame that contains all the labels. 
+        If the dataset is a single tc.Image, this parameter is just a label for
```

Comment:
It seems strange to me to allow this parameter to be either a column name or a class label. Is there precedent for this elsewhere in Turi Create?